### PR TITLE
Update dummies guides

### PR DESCRIPTION
**Pull request by luandrea**
Update dummies guides for tutorial

### CODE REVIEW

File: docs/dheg-build.gradle [hunk 1]
```diff
@@ -34,7 +34,8 @@ buildscript {
 apply plugin: "info.solidsoft.pitest"
 
 pitest {
-  mutationEngine = "descartes"
+  targetClasses = ['myWorld*']
+//  mutationEngine = "descartes"
```
Comment (oscarlvp) [inline docs/dheg-build.gradle]: Wouldn't this mean that descartes is not used?

Comment (luandrea) [reply docs/dheg-build.gradle]: Yes, you are right, but I get errors if I leave that line. 
Do you know how to modify it to run pitest using gradle?

Comment (luandrea) [reply docs/dheg-build.gradle]: Ok, I found another error in the tutorial. I will update it and push it.

**Comment by luandrea** [comment]
The tutorial runs correctly, you can merge it now.

**Pull request merged**